es-vue-base/src/lib-components/EsBreadcrumbs.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <b-breadcrumb
+        v-bind="$attrs"
+        v-on="$listeners" />
+</template>
+<script>
+import { BBreadcrumb } from 'bootstrap-vue';
+
+export default {
+    name: 'EsBreadcrumb',
+    components: {
+        BBreadcrumb,
+    },
+};
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Styles for breadcrumbs seem to be defined in https://github.com/twbs/bootstrap/blob/v4.6.2/scss/_breadcrumb.scss, if you're tweaking bootstrap breadcrumb styles and not building something new / built on-top of, I wonder if these overrides should go in `es-bs-base`

Reply:
Most can; moved in bdf515a